### PR TITLE
Update some Android settings for the sample project

### LINKED ISSUE #10817: Quest requires special manifest entry to enable system keyboard, not documented
## Describe the bug

Not able to enter either textbox in the example scene - Could be related to not having the motion controllers for Quest

## To reproduce

1. Launch Input Field Example scene
2. Observe not being able to interact with textboxes using hand-cursor

## Expected behavior

Can select and interact with textboxes with hand-cursors

## Your setup

- Device Quest
- Unity Version [2020.3.37f1]
- Commit Version acdecca31b97720a6fcda057cc35b3815ad56a02

## Target platform

- OpenVR

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
Add a custom AndroidManifest.xml to enable native keyboard for Quest in our sample Unity project. Also update the Android bundle version to pre.10.

Fixes #10817

### CODE REVIEW

File: UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (keveleigh) [inline UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml:1]: Just to double check, all the build preprocessors that add manifest entries based on OpenXR features still work with a custom manifest, right?

Comment (MaxWang-MS) [reply UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml:1]: I don't have access to a Quest right now, so could @Zee2 or @keveleigh help me to verify this?

Comment (MaxWang-MS) [reply UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml:1]: Also I actually don't know how to tell even if I had a device right now.

Comment (keveleigh) [reply UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml:1]: I can give it a test! The most obvious one would be hand tracking access

Comment (MaxWang-MS) [reply UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml:1]: Thanks! Will be waiting for your results. Please test it with the input field scene to ensure the custom manifest is not being overwritten as well.

Comment (keveleigh) [reply UnityProjects/MRTKDevTemplate/Assets/Plugins/Android/AndroidManifest.xml:1]: Seems good, including keyboard!

**Review by keveleigh — APPROVED** [review]

**Pull request merged**